package.json

```diff
@@ -59,5 +59,5 @@
     "prop-types": "^15.8.1",
     "semver-utils": "^1.1.4"
   },
-  "packageManager": "yarn@3.5.1"
+  "packageManager": "yarn@4.1.0"
```

Comment:
?

Reply:
these changes were introduced by this commit you made: https://github.com/mia-platform/documentation/pull/1187/commits/fbb2bfe3886f261d6defc5a8086474de7b670ad0

Reply:
btw these changes are coherent with main I don't know why github is showing them